src/mouse_events.cpp

```diff
@@ -93,8 +93,15 @@ void mouse_handler::set_side(int side_number)
 
 int mouse_handler::drag_threshold() const
 {
-	// TODO: Use physical screen size.
-	return 14;
+    // Function uses window resolution as an estimate of users perception of distance
+    // Tune this variable if necessary:
+    const unsigned threshold_1080p = 14; // threshold number of pixels for 1080p
+    auto l2_norm = [](unsigned x,unsigned y) {
+        return std::sqrt(std::pow(x,2) + std::pow(y,2));
+    };
```

Comment:
Any reason to not just use `std::hypot`?

Reply:
My own ignorance :)  I have updated now